.github/workflows/sng-ingestor-shared-workflow.yml

```diff
@@ -105,6 +105,16 @@ jobs:
           dockerfile-path: ${{ inputs.dockerfile-path }}
           gcr-image-name: ${{ inputs.gcr-image-name }}
 
+      - name: Notify Slack if docker-build fails
+        uses: extenda/actions/slack-notify@v0
+        if: failure()
+        with:
+          text: |
+            *Build failed for ${{ github.repository }}: ${{ github.workflow }}* :heavy_exclamation_mark:
+            Build failed on ${{ github.event_name }} event. Workflow: ${{ github.workflow }}. Job: ${{github.job}}. Run id: <https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}|${{ github.run_id }}>
+          channel: selfscan-pipeline-notifications
+          service-account-key: ${{ secrets.secret-auth }}
```

Comment:
Please also add this step to the previous job `validate` and adjust the text accordingly, reflecting the failed job.
Something like:
`Build failed for` should be `Validate failed for` when it is part of `validate` job or
`Staging failed for` when it is part of `staging` job, and so on.

Reply:
Done